hardware_interface/include/hardware_interface/joint_state_interface.h

```diff
@@ -41,16 +41,92 @@ namespace hardware_interface
 class JointStateHandle
 {
 public:
-  JointStateHandle() : name_(), pos_(0), vel_(0), eff_(0) {}
+  JointStateHandle() : name_(), pos_(0), vel_(0), eff_(0), absolute_pos_(0), torque_sensor_(0) {}
 
   /**
    * \param name The name of the joint
    * \param pos A pointer to the storage for this joint's position
    * \param vel A pointer to the storage for this joint's velocity
    * \param eff A pointer to the storage for this joint's effort (force or torque)
    */
+
```

Comment:
Extra newline. Also, see comment above about the function comment on the constructors in this file.